### DISCUSSION TITLE
Multilingual Tags

### DISCUSSION THREAD

**Discussion by raymondtri** [area17/twill#1006] [Ideas]
Hello! I am looking into implementing tags in a couple of places, but I can't seem to discern how they are handled with the translatable package. It seems like there isn't any compatibility there. 

Does anyone have a way they have implemented multilingual tags?

**Comment by sercanvirlan** [comment]
I also need this. Anyone knows any hack to implement this ?

**Reply by ifox** [reply]
Hi @sercanvirlan the idea is to create your own tags module which can be translated. The internal tags model is mostly for organizing and filtering in the media library, or very simple use cases. By creating your own module with translations enabled, and a select or browser field where you want user to select tags, you should get what you need and more flexibility.
